src/Source.cpp

```diff
@@ -414,7 +416,7 @@ double SourceSNRDistribution::fz(double z) const{
 }
 
 double SourceSNRDistribution::getAlpha() const {
-	return alpha;
+	return alpha - 1; 
```

Comment:
Can you add a comment here, too? Just to make sure that everybody understands why alpha is used differently than what it was set to.

Reply:
I added the comment